### PR TITLE
Use PrestaShop/php-coding-standards in project

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
https://github.com/PrestaShop/php-coding-standards provides configuration files for using (at that time) PHPStan & PHP-CS-Fixer.

### CODE REVIEW

**Review by PierreRambaud — DISMISSED** [review]

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
   "require": {
     "php": ">=5.6",
     "ext-zip": "*",
-    "symfony/filesystem": "~2.8",
+    "symfony/filesystem": "~3.2",
```
Comment (Progi1984) [inline composer.json:15]: @Quetzacoalt91 Why do you update this package ?

File: tests/phpstan/bootstrap.php [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
 if (!$rootDir) {
     echo '[ERROR] Define _PS_ROOT_DIR_ with the path to PrestaShop folder' . PHP_EOL;
     exit(1);
 }
 
+$pathToModuleRoot = __DIR__ . '/../../';
+
 // Add module composer autoloader
-require_once dirname(__DIR__) . '/../vendor/autoload.php';
+require_once $pathToModuleRoot . 'vendor/autoload.php';
```
Comment (Progi1984) [inline tests/phpstan/bootstrap.php:12]:
```suggestion
 require_once __DIR__ . '/../../vendor/autoload.php'; 
```

File: composer.json (hunk 1 above)
Comment (Quetzacoalt91) [reply composer.json:15]: That was done for composer, in order to match the required version by `prestashop/php-coding-standards`: https://packagist.org/packages/prestashop/php-coding-standards

File: tests/phpstan/bootstrap.php (hunk 2 above)
Comment (Quetzacoalt91) [reply tests/phpstan/bootstrap.php:12]: I aggree with the suggested changes but they can't be applied (the diff is incorrect).

File: composer.json (hunk 1 above)
Comment (Progi1984) [reply composer.json:15]: Ok

File: tests/phpstan/bootstrap.php (hunk 2 above)
Comment (Progi1984) [inline tests/phpstan/bootstrap.php:12]:
```suggestion
require_once __DIR__ . '/../../vendor/autoload.php'; 
```

File: tests/phpstan/bootstrap.php [hunk 3]
```diff
@@ -1,17 +1,20 @@
...
 
 $rootDir = getenv('_PS_ROOT_DIR_');
-
 if (!$rootDir) {
     echo '[ERROR] Define _PS_ROOT_DIR_ with the path to PrestaShop folder' . PHP_EOL;
     exit(1);
 }
 
+$pathToModuleRoot = __DIR__ . '/../../';
+
```
Comment (Progi1984) [inline tests/phpstan/bootstrap.php:10]:
```suggestion
```

File: tests/phpstan/bootstrap.php (hunk 2 above)
Comment (Progi1984) [reply tests/phpstan/bootstrap.php:12]: Fixed with two new suggestions ;)

**Comment by Quetzacoalt91** [comment]
Seems the rework has been done in the meantime. Closing as obsolete.

**Pull request closed without merging**